lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -566,14 +605,18 @@ def generate_clone_spec(src_config)
 
     if config[:customization_ips]
       if get_config(:customization_gw)
-        cust_spec.nicSettingMap = config[:customization_ips].split(',').map { |i| generate_adapter_map(i, get_config(:customization_gw)) }
+        cust_spec.nicSettingMap = config[:customization_ips].split(',').map.with_index { |i, index|
```

Comment:
To give the reader a bit more context, could that `i` variable be something a bit more descriptive like `ip`? `i` is often a loop counter, and we already have `index` so this should reduce some confusion on the part of the reader.